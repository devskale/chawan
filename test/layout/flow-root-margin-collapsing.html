<!DOCTYPE html>
<div style="display: flow-root; background: red; margin-bottom: 1em">
<div style="margin-bottom: 1em">
test
</div>
</div>
test 2
