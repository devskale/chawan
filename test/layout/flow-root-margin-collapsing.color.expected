[48;2;255;0;0mtest                                                                            [49m
[48;2;255;0;0m                                                                                [49m

test 2
